.github/workflows/.test.yml

```diff
@@ -110,6 +110,7 @@ jobs:
           - containerd-rootless
           - containerd-1.6
           - containerd-1.7
+          - containerd-2.0
```

Comment:
containerd 1.6 will EOL in July 2025, so don't worry this list won't grow indefinitely. Once 1.7 EOL's in March 2026; the list should decrement to 2 versions at any time.

Reply:
You can remove 1.6 already